config/locales/en.yml

```diff
@@ -952,6 +953,7 @@ en:
     articles_edited_generic: Pages Edited
     articles_edited_generic_wikidata: Items Edited
     article_scoped_program_info: This is an 'Article Scoped Program', which only counts edits to assigned articles and categories.
+    article_scoped_program_info_wikidata: This is an 'Items Scoped Program', which only counts edits to assigned Items and categories.
```

Comment:
This is the name of the program type. It's not a great name and might change, but we shouldn't make it dynamic based on wikidata vs wikipedia.